resin/auth.py

```diff
@@ -103,6 +125,13 @@ def is_logged_in(self):
         Returns:
             bool: True if logged in, False otherwise.
 
+        Examples:
+            # Check if user logged in.
+            >>> if resin.auth.is_logged_in():
+            ...     print('You are logged in!')
+            ... else:
+            ...     print('You are not logged in!')
```

Comment:
The generated output of this example looks weird, so I suggest you prepend ``>>>`` to the ``# Check if user logged in.`` comment too